src/reducers/lobby.js

```diff
@@ -47,13 +48,29 @@ export default function lobbyReducer(state = initialState(), action) {
 
   case START_NEW_GAME:
     const teams = [RED, BLUE];
-    // make sure we have the right sorts of players
+    let numSpymasters = 0;
```

Comment:
No need to introduce mutable state - you can calculate number of spymasters with `spymastersOf(state.players).length`